src/node.rs

```diff
@@ -919,9 +919,9 @@ pub fn node_loop(
       }
 
       // Receives and handles incoming network messages
-      if tick % 1 == 0 {
-        node_receive_message(&mut node);
-      }
+      // if tick % 1 == 0 {
```

Comment:
I think it is better to leave this if. Yes it is redundant, but it is about following the same structure as the others.